llama_index/indices/base.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         self._storage_context = storage_context or StorageContext.from_defaults()
         self._docstore = self._storage_context.docstore
         self._vector_store = self._storage_context.vector_store
+        self._graph_store = self._storage_context.graph_store
```

Comment:
I don't think the base class should know about graph store?

Reply:
mmm true, but could say the same about the vector store? We can take a look at both in a future PR